src/ossia-max/src/ZeroconfOscqueryListener.cpp

```diff
@@ -11,11 +11,10 @@ namespace ossia
 {
 namespace max
 {
-std::vector<std::shared_ptr<ossia::net::generic_device>> ZeroconfOscqueryListener::m_devices;
```

Comment:
at some point (later in time, it works as it is), wonder if it could be doable to refactor these listeneers which do sensibly the same thing..

Reply:
done in ceb3a6b7